runtime/frequency/src/lib.rs

```diff
@@ -961,17 +962,29 @@ impl pallet_passkey::Config for Runtime {
 	type Currency = Balances;
 }
 
-#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
+#[cfg(any(
+	feature = "frequency",
+	feature = "runtime-benchmarks",
+	feature = "frequency-lint-check",
+))]
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
 /// into the relay chain.
 const UNINCLUDED_SEGMENT_CAPACITY: u32 = 1;
+
+#[cfg(any(not(
```

Comment:
I think this should be `not(any(...))`

Reply:
I think I am going to go back to the original. I find it a little nicer to follow.